src/main/java/nl/techiteasy/Techiteasy_part2/Dto/InputDto/TelevisionInputDto.java

```diff
@@ -1,11 +1,11 @@
 package nl.techiteasy.Techiteasy_part2.Dto.InputDto;
 
-import nl.techiteasy.Techiteasy_part2.Dto.OutputDto.TelevisionOutputDto;
-import nl.techiteasy.Techiteasy_part2.Model.Television;
+import jakarta.validation.constraints.NotNull;
 
 public class TelevisionInputDto {
 
     public Long id;
+    @NotNull
```

Comment:
Dit is de enige validatie die je doet in al je inputDto's. In je ouputdto's gebruik je ook geen validatie, maar daar is dat ook niet nodig